### PR TITLE
Sit

### PR DESCRIPTION
**Pull request by RebeccaStankus**
https://highfidelity.atlassian.net/browse/BUGZ-554

### CODE REVIEW

File: usefulUtilities/sitV2/appResources/appData/sitClientV2.js [hunk 1]
```diff
@@ -370,16 +332,17 @@
...
                 AvatarList.getAvatarsInRange(_this.seatCenterPosition, CAN_SIT_M)
             );
-            // ENABLE JUMP DRIVE KEYS
-            for (var i in REMAPPED_DRIVE_KEYS_DURING_SIT) {
-                MyAvatar.enableDriveKey(REMAPPED_DRIVE_KEYS_DURING_SIT[i]);
+            // Enable movement again
+            for (var i in DISABLED_DRIVE_KEYS_DURING_SIT) {
+                MyAvatar.enableDriveKey(DISABLED_DRIVE_KEYS_DURING_SIT[i]);
             }
+            Controller.actionEvent.disconnect(onActionEvent);
```
Comment (zfox23) [inline usefulUtilities/sitV2/appResources/appData/sitClientV2.js]: Don't you want to do this _before_ re-enabling the drive keys?

Comment (RebeccaStankus) [reply usefulUtilities/sitV2/appResources/appData/sitClientV2.js]: I guess there is an extremely slim chance that the space bar could be hit again. I will move it.

File: usefulUtilities/sitV2/appResources/appData/sitClientV2.js [hunk 2]
```diff
@@ -339,10 +342,11 @@
                 "addAllOtherSittableOverlays",
                 AvatarList.getAvatarsInRange(_this.seatCenterPosition, CAN_SIT_M)
             );
-            
-            // Enable movement again
-            for (var i in DISABLED_DRIVE_KEYS_DURING_SIT) {
-                MyAvatar.enableDriveKey(DISABLED_DRIVE_KEYS_DURING_SIT[i]);
+            if (settingsEntityID === _this.entityID) {
```
Comment (zfox23) [inline usefulUtilities/sitV2/appResources/appData/sitClientV2.js]: this concerns me a bit but i know there will be bugs with this implementation, so let's get this out there first!

Comment (RebeccaStankus) [reply usefulUtilities/sitV2/appResources/appData/sitClientV2.js]: yeah, there is def an issue around changing seats right now and I think that will need to change but it fixes the more urgent problem with seat floating.

**Pull request merged**